client/components/routes/Routes.jsx

```diff
@@ -12,6 +12,10 @@ function Routes() {
   const [criteria, setCriteria] = useState(false);
   const [user, setUser] = useState({});
   const [loginOpen, setLoginOpen] = useState(false);
+  const [loggedInUser, setLoggedInUser] = useState({
+    id: 1,
```

Comment:
assuming that the user info from Helen's component will be passed here.

Reply:
Yeah, hoping she sends that down.